webmud/server.py

```diff
@@ -0,0 +1,43 @@
+import BaseHTTPServer
+import json
+import time
+
+class WebMUDRequestHandler(BaseHTTPServer.BaseHTTPRequestHandler):
+    def do_GET(self):
+        # Eventually be picky about the endpoint, but right now accept any request.
+        # (This is because eventually the mud can serve the client HTML/CSS/JS too!)
+        self.send_response(200)
+        self.send_header('Content-type', 'application/json')
+        self.send_header('Access-Control-Allow-Origin', '*')
+        self.end_headers()
+        response = []
+        response.append(self._console_out("Ok @ {}".format(time.time())))
+        response.append(self._console_in_flash(200, [["#F00", "#0F0", "#00F"][int(time.time())%3], "#FFF"]))
```

Comment:
I just wanted to demonstrate something dynamic but checkable from the server.

Takes the current epoch second mod three. 0 is red, 1 is green, 2 is blue.